package.json

```diff
@@ -4,15 +4,20 @@
   "description": "Javascript library for DHIS2",
   "main": "d2.js",
   "scripts": {
```

Comment:
I can see `npm` calls. Are we not using yarn for this one? (I just don't know, just want to raise a question mark here)

Reply:
This is also because that's how it was before, but it seems fine to me. Whoever has yarn most likely has npm, but the other way around is less likely I'd say..

Reply:
Fair enough!